src/main/java/com/michelin/ns4kafka/services/clients/schema/entities/TopicDescriptionUpdateResponse.java

```diff
@@ -0,0 +1,15 @@
+package com.michelin.ns4kafka.services.clients.schema.entities;
+
+import lombok.Builder;
+import org.apache.avro.data.Json;
+
+/**
+ * Update topic description response.
+ *
+ * @param mutatedEntities   The updated entities
+ *
+ */
+@Builder
+public record TopicDescriptionUpdateResponse(Json mutatedEntities) {
```

Comment:
Not sure we should use this pseudo-json type. An actual POJO would be better, but I've seen the signature of the response is wide

Do you actually use the response of the `updateDescription` endpoint ? If no, remove this class for now and just make it return a `Mono<Void>` or `Mono<HttpResponse<Void>>`

Reply:
The http response is used for subscription, so it is not possible to have `Mono<Void>`. I tested and `Mono<HttpResponse<Void>>` doesn't work either because the update request returns non null response so the client has trouble to deserialize the response

So I made an actual POJO without the Json type